recipes/capnproto/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set version = "0.5.3" %}
+
+package:
+  name: capnproto
+  version: {{ version }}
+
+source:
+  fn: v{{ version }}.tar.gz
+  url: https://github.com/sandstorm-io/capnproto/archive/v{{ version }}.tar.gz
+  md5: 909bd13ad6b8bc840ac78ab8f5bcb0a4
+
+build:
+  number: 0
+  skip: True  # [win and py<35]
+  features:
+    - vc14  # [win and py35]
+
+requirements:
+  build:
+    - cmake
+  run:
+
+test:
```

Comment:
We need to require `python` as a test dependency on Windows only. I can't remember right now whether this was a bug in `conda-build` or a feature due to its accommodation of unusual but important use cases.